CommonLibSF/src/SFSE/Logger.cpp

```diff
@@ -21,5 +24,31 @@ namespace SFSE
 			path /= "My Games\\Starfield\\SFSE\\Logs";
 			return path;
 		}
+
+		void init()
+		{
+			auto path = log_directory();
+			if (!path)
+				return;
+			
+			const auto data = PluginVersionData::GetSingleton();
+			*path /= std::format("{}.log", data->GetPluginName());
```

Comment:
why not `fmt::format`?

Reply:
I'm going to slowly switch things from fmt to std as time goes on.

Reply:
Why is that?

Reply:
To eventually remove fmt as a dependency. The only thing blocking me from doing it all completely is that the vcpkg port of spdlog doesn't support turning on spdlog's own `std::format` support.

Reply:
As I understand it the standard library implementation is worse--is there any reason to move off fmt as a dependency?

Reply:
How is it worse? It has some missing features, but several have been addressed in c++23 and more will come in the next standard version. It is based on `fmt` with the author being involved in it's development.

Reply:
C++26 isn't gonna be around anytime soon, I don't see any reason to move off fmt at the moment

Reply:
There's no reason not to. This is all internal to the library. users can still use fmt if they'd like.

Reply:
It seems like an entirely unnecessary change, it isn't ideal to do it piecemeal either imo. I vote we stick with fmt until there's a strong reason to use something else

Reply:
We're already including `<format>`. I'm going to use it.

Reply:
I don't think this is a useful way to resolve a disagreement in a collaborative project, I'll set up a poll and we can see what everyone thinks